<launch>
  <rosparam command="load" file="$(find protobot_hardware)/config/LED_settings.yaml"/>
  <node name="protobot_hardware_LED_array_server_node" pkg="protobot_hardware" type="protobot_hardware_LED_array_server_node" output="screen"/>
</launch>
